configs/common/packages.yaml

```diff
@@ -111,7 +111,7 @@
       version: [1.6.37]
     # Attention - when updating also check discover site config
     mapl:
-      version: [2.12.3]
+      version: [2.20.0]
```

Comment:
Hopefully this builds on macOS ...

Reply:
Crap, this is the wrong version

Reply:
Glad that I asked ;-) Let's see what the CI builds say.